### PR TITLE
create catalog

### PR DESCRIPTION
**Pull request by Goto-Jesus**
- [DEMO LINK](https://Goto-Jesus.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Goto-Jesus.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/utils/container.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.container {
+  min-width: 1024px;
+  max-width: 1200px;
+
+  margin: 0 auto;
+  padding: 0 50px;
+  // border: 1px solid black;
```
Comment (jstmpelowycz) [inline src/styles/utils/container.scss:7]: remove commented code

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,391 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container header__container">
+        <a href="#home" class="logo"></a>
```
Comment (jstmpelowycz) [inline src/index.html:15]: should link be a separate BEM-block?

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,391 @@
...
+            </div>
+  
+            <p class="card__amount-of-reviews">Reviews: 5</p>
+          </div>
+  
+          <div class="card__price-block">
+            <p class="card__text">Price:</p>
+  
+            <p class="card__price">$2,199</p>
+          </div>
```
Comment (jstmpelowycz) [inline src/index.html:111]:
```suggestion
          <div class="card__price">
            <p class="card__price-label">Price:</p>
  
            <p class="card__price-value">$2,199</p>
          </div>
```

File: src/styles/blocks/header.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+  }
+
+  &__item {
+    display: inline-block;
+    position: relative;
+    margin: 0 0 0 20px;
```
Comment (jstmpelowycz) [inline src/styles/blocks/header.scss:29]: better to reset it in `reset.scss`, what do you think? 🤔

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  &__item:first-child {
+    margin: 0;
+  }
```
Comment (jstmpelowycz) [inline src/styles/blocks/header.scss:34]: let's use `:not()`

File: src/styles/blocks/header.scss [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    justify-content: space-between;
+    align-items: center;
+  }
+}
+
+.logo {
+  background-image: url("../images/logo.png");
+  width: 40px;
+  height: 40px;
+}
```
Comment (jstmpelowycz) [inline src/styles/blocks/header.scss:17]: make it a BEM-element and use nesting to assign to `header`

File: src/styles/blocks/header.scss [hunk 7]
```diff
@@ -0,0 +1,65 @@
+
+.header {
```
Comment (jstmpelowycz) [inline src/styles/blocks/header.scss:2]:
```suggestion
.header {
```

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,109 @@
+/* #region card */
+.main {
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding: 0 40px;
+    padding-top: 50px;
```
Comment (jstmpelowycz) [inline src/styles/blocks/card.scss:8]:
```suggestion
    padding: 50px 40px 0;
```

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+    max-width: 1024px;
```
Comment (jstmpelowycz) [inline src/styles/blocks/card.scss:10]: the checklist, p.13

File: src/styles/blocks/card.scss [hunk 10]
```diff
@@ -0,0 +1,109 @@
+/* #region card */
+.main {
```
Comment (jstmpelowycz) [inline src/styles/blocks/card.scss:2]: too many BEM-blocks per file

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
let's fix failing tests (font issue)
Screenshot 2023-03-02 at 10 56 39

take care of your code, see the comments 👇

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
testing table sync

**Review by jstmpelowycz — COMMENTED** [review]
let's add header box-shadow as in the mockup
Screenshot 2023-03-02 at 11 57 29

File: src/styles/utils/container.scss (hunk 1 above)
Comment (Goto-Jesus) [reply src/styles/utils/container.scss:7]: +

File: src/styles/blocks/header.scss (hunk 5 above)
Comment (Goto-Jesus) [reply src/styles/blocks/header.scss:34]: done thx

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: white;
+
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```
[the remark this answers is not in the document]
Comment (Goto-Jesus) [reply src/styles/blocks/card.scss:25]: done

**Review by liliia-livandovskaya — APPROVED** [review]

**Pull request closed without merging**